SwaprHQ/stackly-ui
# [STK-89][FEAT] - TokenPicker token select functionality

## Fixes: [STK-89](https://linear.app/swaprdev/issue/STK-89/tokenpicker-token-select-functionality)

### Description

* Assets import/export refactor
* Implemented token selection in the `TokenPicker` to be displayed in `Stackbox`

![image](https://github.com/SwaprHQ/stackly-ui/assets/21271189/108f01c0-bb73-44f6-b0d2-2b983c67e12f)

### Preview
https://feat-stk-89-tokenpicker-toke.stackly-ui.pages.dev/

### How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Stackly landing page, click any "Select token" button and wait for the modal to pop-up
4) The TokenPicker view should select a token, close the modal and clear the `SearchBar` only if it has a written query on any:
4.1)`TokenList` interaction
4.2)`CommonTokens` interaction
